src/scripts/assets/covid_lab_results_standard.json

```diff
@@ -0,0 +1,18 @@
+{
+    "POSITIVE": [
+        "POSITIVE",
+        "DETECTED",
+        "REACTIVE"
+    ],
+    "NEGATIVE": [
+        "NEGATIVE",
+        "NOT DETECTED",
+        "NON-REACTIVE",
+        "(NOTE) NOT DETECTED"
+    ],
+    "INCONCLUSIVE": [
+        "INCONCLUSIVE",
+        "INVALID",
+        "INVALID RESULT"
+    ]
+}
```

Comment:
nitpick: newline

Reply:
I'm not sure I follow you here. You would like me to add newline (#19)?

Reply:
(my guess is yes? Typically when github shows that minus sign with a circle it's mad that there isn't a trailing newline)